mslib/utils/mssautoplot.py

```diff
@@ -28,19 +28,26 @@
 from datetime import datetime, timedelta
 import io
 import logging
+import re
 import os
 import sys
+import requests
+from urllib.parse import urljoin
+import json
+from PyQt5.QtCore import Qt
 
```

Comment:
later imports should be sorted later

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.